packages/data-stores/src/launchpad-navigator/actions.ts

```diff
@@ -0,0 +1,49 @@
+import apiFetch from '@wordpress/api-fetch';
+import { select } from '@wordpress/data';
+import { canAccessWpcomApis } from 'wpcom-proxy-request';
+import { GeneratorReturnType } from '../mapped-types';
```

Comment:
Should this be a `type` import?
```suggestion
import type { GeneratorReturnType } from '../mapped-types';
```